### DISCUSSION TITLE
Generated reveal.js html formats differently on different devices.

### DISCUSSION THREAD

**Discussion by rpace314159** [Zettlr/Zettlr#2794] [Q&A]
I have a ThinkPad laptop (Windows 10) and a HP Desktop (Linux SuSE), with a shared external monitor between the two via a KVM switch.  I wrote a document using Zettlr and exported it using the reveal.js export type.  From my Desktop, Firefox displays the document as expected...When I display the document from my laptop (on the external monitor), some of the slides do not display correctly...on one slide some of the content is shifted off the page.  Two slides, the # and ## headers overlap.  I was attempting to avoid using Powerpoint. The Firefox settings are the same except a different font type, but same size.  Both Firefox settings allow pages to use their own fonts and sizes. I do use some embedded HTML in the document, in particular <div> boxes to allow text to flow around images.  As I said, it displays properly from my desktop Firefox. Fonts and images load just a little bit bigger from my laptop.  Maybe I need the same sans-serif font family on Windows 10, that I use on Linux.  On Linux I have the Dejavu  family, which I don't have on my Windows 10. 

I appreciate any suggestions anyone might have.

**Comment by nathanlesage** [comment]
Does this problem also happen if you use revealJS slides "manually" (that is, without the Pandoc exporting from Zettlr"?

**Reply by rpace314159** [reply]
Not sure I know what you mean.  I don't have reveal.js installed.  Are you suggesting I install reveal.js and try my md source directly with reveal.js?

**Reply by nathanlesage** [reply]
You cannot "install" reveal.js; all I meant was whether or not this also happens if you create such a presentation manually (either via using Pandoc directly or via manually writing such an HTML file)
